### PR TITLE
Feat: Home

### PR DESCRIPTION
**Pull request by dikshadhote**
Following features are added
1. Header footer
2. Banner image
3. Top categories 

preview: https://deploy-preview-3--dreamstore-home.netlify.app/

image
image

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+import { Link } from "react-router-dom";
+import "./../../App.css";
+import axios from "axios";
 export default function Home() {
+  const [categoriesData, setCategoriesData] = useState([]);
+  async function apiCall() {
+    const { data } = await axios.get("/api/categories");
+    const { categories } = data;
+    setCategoriesData(categories);
+    console.log(categories);
```
Comment (anuj-kumary) [inline src/components/Home/Home.js]: Don't give unwanted comments.

File: src/components/Home/Home.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.category-card {
+  height: 17.87rem;
+}
```
Comment (samyakshah3008) [inline src/components/Home/Home.css:3]: You can prefer giving min or max height for responsiveness(can add later on)

File: src/components/Home/Home.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+.carousel-banner {
+  background-color: var(--card-border-color);
+  margin: 1rem;
+  border-radius: 5px;
+}
+
+img.banner-img {
+  opacity: 0.8;
+  border-radius: 5px 0 0 5px;
```
Comment (samyakshah3008) [inline src/components/Home/Home.css:18]: Good use of shorthand property

File: src/components/Home/Home.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  background-color: var(--orange-color);
+  position: absolute;
+  bottom: 45rem;
+}
+.card-category-txt {
+  position: absolute;
+  bottom: 1rem;
+  padding: 10px;
+  border-radius: 64px;
+  background-color: white;
```
Comment (samyakshah3008) [inline src/components/Home/Home.css]: You can use css variables
and also prefer rem unit when it is > 10px

**Review by anuj-kumary — APPROVED** [review]
Good work, good naming conventions and clean code

**Review by samyakshah3008 — APPROVED** [review]
Good folder structure 
Loved your UI 
Liked that how you gave preview description, lead to smooth review 

Gave few suggestions by single comments, you can refactor later on 

Rest is all good for approval ✅

**Pull request merged**